workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/pages/AllScreens.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayLayout from '@clayui/layout';
+import {ClayVerticalNav} from '@clayui/nav';
+import {ReactNode} from 'react';
+
+import {useHash} from '../hooks/useHash';
+import {ScreenType} from '../types';
+import TicketsDashboard from './TicketsDashboard';
+import TicketsOverview from './TicketsOverview';
+
+const ROUTES = [
+	{
+		element: <TicketsDashboard screenType={ScreenType.INTEGRATED} />,
+		href: '#dashboard',
+		label: 'Dashboard',
+	},
+	{
+		element: <TicketsOverview />,
+		href: '#overview',
+		label: 'Tickets Overview',
+	},
+];
+
+const HREF_COMPONENT_MAP: {[key: string]: ReactNode} = {};
+
+ROUTES.forEach((route) => {
+	HREF_COMPONENT_MAP[route.href] = route.element;
+});
+
+const AllScreens: React.FC = () => {
+	const {hash} = useHash('#dashboard');
+
+	return (
+		<ClayLayout.ContentRow padded>
+			<ClayLayout.ContentCol>
+				<div className="mb-2 text-uppercase">Site</div>
+
+				<ClayVerticalNav active={hash} items={ROUTES} large={false}>
+					{(item: any) => (
```

Comment:
Try not to use `any` type, it'll just lead to bugs

Reply:
https://liferay.slack.com/archives/C049XG98GQL/p1706010034494419